From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ximin Luo <infinity0@pwned.gg>
Date: Sat, 13 Jun 2026 10:10:44 +0100
Subject: [PATCH 2/2] Extend production workarounds to userdebug as well

---
 scripts/gen_build_prop.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/gen_build_prop.py b/scripts/gen_build_prop.py
index b53885fc8..27957351f 100644
--- a/scripts/gen_build_prop.py
+++ b/scripts/gen_build_prop.py
@@ -384,7 +384,7 @@ def append_additional_system_props(args):
 
   props.append("ro.control_privapp_permissions=enforce")
   props.append("net.tethering.noprovisioning=true")
-  if config["BuildVariant"] == "user":
+  if config["BuildVariant"] == "user" or config["BuildVariant"] == "userdebug":
     # at least one app (Revolut) refuses to work with yellow verifiedbootstate
     props.append("ro.appcompat_override.ro.boot.verifiedbootstate=green")
 
-- 
2.54.0

