From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andrew Gunnerson <accounts+github@chiller3.com>
Date: Thu, 3 Jul 2025 15:16:27 -0400
Subject: [PATCH] Keep external ports disabled at boot on userdebug builds

---
 init/Android.bp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/init/Android.bp b/init/Android.bp
index 6f50c0900..1cf21f61c 100644
--- a/init/Android.bp
+++ b/init/Android.bp
@@ -139,9 +139,6 @@ libinit_cc_defaults {
     product_variables: {
         debuggable: {
             cppflags: [
-                "-UDISABLE_EXTERNAL_PORTS_ON_NORMAL_BOOT",
-                // USB adb access is needed for debugging early boot failures
-                "-DDISABLE_EXTERNAL_PORTS_ON_NORMAL_BOOT=0",
                 "-UALLOW_FIRST_STAGE_CONSOLE",
                 "-DALLOW_FIRST_STAGE_CONSOLE=1",
                 "-UALLOW_LOCAL_PROP_OVERRIDE",
@@ -451,10 +448,6 @@ init_first_stage_cc_defaults {
     product_variables: {
         debuggable: {
             cflags: [
-                "-UDISABLE_EXTERNAL_PORTS_ON_NORMAL_BOOT",
-                // USB adb access is needed for debugging early boot failures
-                "-DDISABLE_EXTERNAL_PORTS_ON_NORMAL_BOOT=0",
-
                 "-UALLOW_FIRST_STAGE_CONSOLE",
                 "-DALLOW_FIRST_STAGE_CONSOLE=1",
 
-- 
2.52.0

